apps/webapp/app/services/slack.server.ts

```diff
@@ -0,0 +1,40 @@
+import { WebClient } from "@slack/web-api";
+import { env } from "~/env.server";
+import { logger } from "./logger.server";
+
+const slack = new WebClient(env.SLACK_BOT_TOKEN);
+
+type SendNewOrgMessageParams = {
+  orgName: string;
+  whyUseUs: string;
+  userEmail: string;
+};
+
+export async function sendNewOrgMessage({ orgName, whyUseUs, userEmail }: SendNewOrgMessageParams) {
+  try {
```

Comment:
If either of the environment variables aren't set then we would just return here.